presto-docker/etc/node.properties

```diff
@@ -0,0 +1,4 @@
+node.environment=docker
+node.data-dir=/data/presto
+catalog.config-dir=/usr/lib/presto/etc/catalog
```

Comment:
We shouldn't need to configure this, as the launcher should start the server in the config directory. If that doesn't happen, various config things will not work correctly.

Reply:
To be precise, launcher starts the server in the datadir and creates etc link there to the etcdir.

Reply:
👍